Assets/AnkrSDK/Runtime/MirageAPI/MirageID/Implementation/MirageIdUserRequests.cs

```diff
@@ -9,10 +9,21 @@ namespace AnkrSDK.MirageAPI.MirageID.Implementation
 {
 	public class MirageIdUserRequests : IMirageIdUserRequests
 	{
+		private IMirageIdEnv _env;
 		private string _clientId;
 		private string _clientSecret;
 		private string _userToken;
 
+		public MirageIdUserRequests()
+		{
+			_env = new MirageIdEnv();
```

Comment:
there is no reason to create it each time - these env classes contain a state and each time you will change envsettings you will have to do it in every instance, which is quite uncomfortable.